src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Web/Components/NotFoundComponent.razor.scss

```diff
@@ -0,0 +1,171 @@
+﻿@import '../Styles/_shared.scss';
+
+.main {
+    display: flex;
+    flex-flow: column nowrap;
+    justify-content: space-evenly;
+    align-items: center;
+    width: 100%;
+    height: 100%;
+}
+
+.cloud-1 {
+    background-image: url('/images/cloud.svg');
+    background-size: cover;
+    background-position: center;
+    background-repeat: no-repeat;
+    width: 88px;
```

Comment:
I think you should use the rem function for all the places where you have entered the value in pixels.
@maryamhdr